### ISSUE TITLE
Add new automod fields

### ISSUE THREAD

**Issue by Skelmis** [nextcord/nextcord#1039]
### Summary

https://github.com/discord/discord-api-docs/pull/5778

### What is the feature request for?

The core library

### The Problem

Implement https://github.com/discord/discord-api-docs/pull/5778 once it is merged into the documentaiton

### The Ideal Solution

Implement the new feature
